homeworks/01-bubble-sort/src/components/column/Column.tsx

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from "react";
+import './index.css'
+
+type ColumnProps = {
+	size: number
+}
+
+class Column extends Component<ColumnProps, {}> {
```

Comment:
This should be a Function (Stateless) Component

Reply:
fixed